ibis/backends/tests/conftest.py

```diff
@@ -90,7 +90,7 @@ def pytest_runtest_call(item):
             feature for feature in features if not getattr(backend, feature)
         ]
         if missing_features:
-            pytest.mark.skip(
```

Comment:
actually you should add: `item.add_marker`

Reply:
Hmm I'm not able to get `item.add_marker` + `pytest.mark.skip` working, even with some variations that work when I simply replace `skip` with `xfail`. I'm not sure what the issue is with dynamically adding `pytest.mark.skip`.

However despite the current version with `pytest.skip` not fixing any existing tests in the CI, I can confirm that the current version does work (locally added a test that fails and confirms it now skips). `pytest.skip` is also what theother custom "skip" markers use ([example](https://github.com/timothydijamco/ibis/blob/019ccd5122befd3b843b32abf8b59991244596c9/ibis/backends/tests/conftest.py#L85)) so perhaps the issue with `pytest.mark.skip` had been run into before